### PR TITLE
Verify proper NTP installation on Cassandra hosts

### PR DESCRIPTION
**Pull request by tiago-loureiro**
Too often we have been bitten by this; this adds a final check on the cassandra playbook that NTP is _indeed_ properly setup. Furthermore, it adds a simple playbook to verify cassandra hosts when necessary. Can be adapted/adjusted to run on all hosts, and potentially even added to https://github.com/geerlingguy/ansible-role-ntp

### CODE REVIEW

**Comment by fisx** [comment]
when this goes to develop / prod, don't forget to update [this link](https://github.com/wireapp/wire-docs/blame/952df7d7b127047b34dd6488d90a2cfed1b49eb2/src/how-to/install/troubleshooting.rst#L63)

File: ansible/roles/ntp-verify/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# "     remote           refid      st t when poll reach   delay   offset  jitter",
+# "==============================================================================",
+# " 0.ubuntu.pool.n .POOL.          16 p    -   64    0    0.000    0.000   0.000",
+#
+# ^-- This is an example of a "bad" output, where no NTP servers can actually be reached
+#     The check is a bit fragile but may help catching things earlier on
+- fail:
+    msg: You might have a misconfigured check the output above and ensure that you are
+         able to connect to the configured NTP servers! If you have no idea what this
+         means, refer to https://detailed.wordpress.com/2017/10/22/understanding-ntpq-output
```
Comment (fisx) [inline ansible/roles/ntp-verify/tasks/main.yml]:
```suggestion
    msg: Check the output above and ensure that you are able to connect to the configured
         NTP servers! If you have no idea what this means, refer to
         https://detailed.wordpress.com/2017/10/22/understanding-ntpq-output
```

**Comment by jschaul** [comment]
Nice one! It might be useful to create an own git repository and move this new role there, so it can more easily be imported in other places where we use ansible.

**Comment by tiago-loureiro** [comment]
> Nice one! It might be useful to create an own git repository and move this new role there, so it can more easily be imported in other places where we use ansible.

Sounds good, created https://github.com/wireapp/ansible-ntp-verify and adjusted this one.

**Comment by jschaul** [comment]
> git push
> ERROR: Permission to wireapp/ansible-ntp-verify.git denied to jschaul.

(please give everyone access to that repo)

**Comment by tiago-loureiro** [comment]
> > git push
> > ERROR: Permission to wireapp/ansible-ntp-verify.git denied to jschaul.
> 
> (please give everyone access to that repo)

Argh! Done; still, this is a public repo so I'm puzzled: would plain https://... links just work? Not sure about [the docs](https://docs.ansible.com/ansible/latest/galaxy/user_guide.html)

**Comment by jschaul** [comment]
> Argh! Done; still, this is a public repo so I'm puzzled: would plain https://... links just work? Not sure about [the docs](https://docs.ansible.com/ansible/latest/galaxy/user_guide.html)

This was about *push* access to the git repo for team members (which now works), which is needed to make changes and PRs without forking. Yes of course every public repo can be cloned and read by everyone, also galaxy, even without granting anyone push access.

**Pull request merged**